app/controllers/sessions_controller.rb

```diff
@@ -2,12 +2,16 @@ class SessionsController < ApplicationController
   def new; end
 
   def create
-    user = User.find_by email: params[:session][:email].downcase
+    user = User.find_by(email: params[:session][:email].downcase)
     if user&.authenticate(params[:session][:password]).present?
       flash[:success] = t ".login_success"
       log_in user
-      params[:session][:remember_me] == Settings.checkbox ? remember(user) : forget(user)
-      redirect_to user
+      params[:session][:remember_me] = if Settings.checkbox
```

Comment:
toán tử 3 ngôi chỗ này k dc à e

Reply:
toán tử 3 ngôi rubocop nó báo là phải dùng if else a ạ